next.config.js

```diff
@@ -9,7 +9,7 @@ const moduleExports = {
     ignoreBuildErrors: false,
   },
 
-  target: 'serverless',
+  target: 'experimental-serverless-trace',
```

Comment:
fixes the following bug, as suggested by https://github.com/getsentry/sentry-javascript/issues/3575#issuecomment-848954576

![Screen Shot 2022-08-05 at 11 43 33 PM](https://user-images.githubusercontent.com/12162433/183233375-a8be5bd2-10b3-4249-9130-b14b6fc3e77e.png)

doesn't seem like it'll be a dealbreaker: https://serverless-nextjs.com/docs/inputs/